modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-10-0_{context}"]
+= Release notes for {gitops-title} 1.10.0
+
+{gitops-title} 1.10.0 is now available on {product-title} 4.12 and 4.13.
```

Comment:
This should also include 4.14

Reply:
@reginapizza 4.14 is not yet GA. we will have to have a 1 pointer JIRA issue to change it here later after 4.14 is out and LIVE.

Reply:
Noted!